### PR TITLE
[Merged by Bors] - Update to spec v1.1.8

### PR DESCRIPTION
**Pull request by michaelsproul**
## Proposed Changes

Change the canonical fork name for the merge to Bellatrix. Keep other merge naming the same to avoid churn.

I've also fixed and enabled the `fork` and `transition` tests for Bellatrix, and the v1.1.7 fork choice tests.

Additionally, the `BellatrixPreset` has been added with tests. It gets served via the `/config/spec` API endpoint along with the other presets.

### CODE REVIEW

File: testing/ef_tests/check_all_files_accessed.py [hunk 1]
```diff
@@ -30,18 +30,11 @@
...
-    "tests/mainnet/altair/merkle/single_proof",
-    "tests/minimal/altair/merkle/single_proof",
-    "tests/mainnet/merge/merkle/single_proof",
-    "tests/minimal/merge/merkle/single_proof",
-    # FIXME(merge): Merge transition tests are  now available but not yet passing
-    "tests/mainnet/merge/transition/",
-    "tests/minimal/merge/transition/",
+    "tests/.*/.*/merkle/single_proof",
+    # One of the EF researchers likes to pack the tarballs on a Mac
+    ".*\.DS_Store.*"
```
Comment (realbigsean) [inline testing/ef_tests/check_all_files_accessed.py:37]: :joy:

**Review by realbigsean — APPROVED** [review]
Looks good! Good call combining the fork choice EF test handlers

**Comment by michaelsproul** [comment]
There are some more renames to the presets and constants that I need to push before this will be ready. I'll get to that tomorrow

**Comment by michaelsproul** [comment]
This is ready for review, but blocked on merging until Pari gives the signal :point_right:

**Comment by parithosh** [comment]
Could this PR be merged so I can go ahead with deploying the update? (Need the image from your CI :D)

**Review by realbigsean — APPROVED** [review]
LGTM

**Comment by realbigsean** [comment]
@parithosh I think we're gonna hold off on merging this until lighthouse v2.1 is released, which might be later this week.  I could push an image for this branch to my docker repo, or if you want to wait until later tonight we could manually push an image to the sigp docker repo?  Do either of these work for you?

**Comment by parithosh** [comment]
It'd be great if you could push an image to your docker repo, that should be enough for now. I can post a message in some discord forums to let people know about the short term mismatch.

**Comment by michaelsproul** [comment]
Merging this in advance of v2.1.0. Docker images should be out shortly after this hits `unstable`

bors r+

**Pull request closed without merging**